modules/Robot.h

```diff
@@ -24,17 +24,20 @@ class Robot {
    */
   Robot();
 
+  // DI 用（カメラを差し替えたいテストで使用）
+  Robot(ICameraCapture& cam);
+
   /**
    * @brief MotorControllerのインスタンスの参照を返す
    * @return メンバ変数motorController(MotorControllerのインスタンス)の参照
    */
   MotorController& getMotorControllerInstance();
 
   /**
-   * @brief CameraCaptureのインスタンスの参照を返す
+   * @brief ICameraCaptureのインスタンスの参照を返す
    * @return メンバ変数cameraCapture(CameraCaptureのインスタンス)の参照
```

Comment:
ICameraCaptureのインスタンスの参照を返すなら、こうかな？
```suggestion
   * @return メンバ変数cameraCapturePtr（ICameraCaptureの実装インスタンス）への参照
```